src/armTemplates/ase.ts

```diff
@@ -0,0 +1,31 @@
+import functionApp from "./resources/functionApp.json";
+import appInsights from "./resources/appInsights.json";
+import storage from "./resources/storageAccount.json";
+import appServicePlan from "./resources/appServicePlan.json";
+import hostingEnvironment from "./resources/hostingEnvironment.json";
+
+export function generate() {
```

Comment:
These are examples of uber templates that merge in the required resources and parameters